backend/src/utils/http.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express'
+import { ByteRange } from '@autonomys/file-caching'
+
+export const getByteRange = (req: Request): ByteRange | undefined => {
+  const byteRange = req.headers['content-range']
```

Comment:
I think this should be looking for a [`range` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Range_requests) in the request. [`content-range`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Range) is used in the response.